source/contracts/reporting/DisputeBondToken.sol

```diff
@@ -95,6 +95,23 @@ contract DisputeBondToken is DelegationTarget, ITyped, Initializable, ERC20Basic
         return _amountToMint;
     }
 
+    function withdrawInEmergency() public onlyInBadTimes returns (bool) {
+        require(msg.sender == bondHolder);
+        IReputationToken _reputationToken = getReputationToken();
+        uint256 _amountToTransfer = _reputationToken.balanceOf(this);
+        if (bondRemainingToBePaidOut > bondAmount) {
+            uint256 _amountToCollectFeesOn = _amountToTransfer.min(bondRemainingToBePaidOut - bondAmount);
+            market.getReportingWindow().collectReportingFeesInEmergency(bondHolder, _amountToCollectFeesOn);
```

Comment:
The escape hatch code should be as simple as possible, and for this reason I don't think we should try to distribute fees to stakers.  I think we should just let the various tokens be turned back into REP and nothing more.  We can address the problem of "what to do with the fees" separately.

At a glance, I think this withdraw function would be ~4 lines long and have no conditionals if we don't try to payout fees.